### PR TITLE
Repaths plant analyzers

### PR DESCRIPTION
**Pull request by PKPenguin321**
So they are no longer children of regular analyzers

Fixes https://github.com/tgstation/tgstation/issues/17039

To be honest I don't know if this will work since I just mass find-replaced, but from what I skimmed over it should be fine

### CODE REVIEW

**Comment by Core0verload** [comment]
Seems like you forgot to update the maps.

**Comment by PKPenguin321** [comment]
Ah, yeah the findreplace didn't touch map files

Will do some time tomorrow

**Comment by Jordie0608** [comment]
If you weren't already, I suggest find/replacing the maps with a text editor to avoid needing to mapmerge them.

**Comment by PKPenguin321** [comment]
okay, so assuming travis passes i request that this gets merged ASAP to prevent conflicts, since it touches 15 files now

File: code/modules/hydroponics/hydroitemdefines.dm [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+/obj/item/device/plant_analyzer
 	name = "plant analyzer"
 	desc = "A scanner used to evaluate a plant's various areas of growth."
 	icon = 'icons/obj/device.dmi'
 	icon_state = "hydro"
 	item_state = "analyzer"
 	origin_tech = "magnets=1;biotech=1"
 
-/obj/item/device/analyzer/plant_analyzer/attack_self(mob/user)
+/obj/item/device/plant_analyzer/attack_self(mob/user)
```
Comment (Jordie0608) [inline code/modules/hydroponics/hydroitemdefines.dm]: This proc can be removed now that it doesn't have to override the parent behavior.

Comment (PKPenguin321) [inline code/modules/hydroponics/hydroitemdefines.dm]: reee ok

**Comment by PKPenguin321** [comment]
once travis is done somebody please merge

**Pull request merged**